src/io/flutter/logging/FlutterConsoleLogManager.java

```diff
@@ -6,62 +6,246 @@
+
+      // TODO(devoncarew): send analytics for the diagnosticsNode
+
+      if (SHOW_STRUCTURED_ERRORS) {
+        queue.add(() -> {
+          try {
+            processFlutterErrorEvent(diagnosticsNode);
+          }
+          catch (Throwable t) {
+            LOG.warn(t);
+          }
+        });
+      }
+    }
+    catch (Throwable t) {
+      LOG.warn(t);
+    }
+  }
+
+  private static final String errorSeparator =
+    "◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤";
+
+  /**
+   * Pretty print the error using the available console syling attributes.
+   */
+  private void processFlutterErrorEvent(@NotNull DiagnosticsNode diagnosticsNode) {
+    final String description = " " + diagnosticsNode.toString() + " ";
+
+    final int targetLength = errorSeparator.length();
+    final String prefix = StringUtil.repeat("◢◤", (targetLength - description.length()) / 4);
+    final String suffix = StringUtil.repeat("◢◤", ((targetLength - (description.length() + prefix.length())) / 2));
+
+    console.print("\n" + prefix + description + suffix + "\n", TITLE_CONTENT_TYPE);
+
+    DiagnosticLevel lastLevel = null;
+
+    // TODO(devoncarew): Create a hyperlink to a widget - ala 'widget://inspector-1347'.
+
+    for (DiagnosticsNode property : diagnosticsNode.getInlineProperties()) {
+      // Add blank line between hint and non-hint properties.
+      if (lastLevel != null && lastLevel != property.getLevel()) {
+        if (lastLevel == DiagnosticLevel.hint || property.getLevel() == DiagnosticLevel.hint) {
+          console.print("\n", NORMAL_CONTENT_TYPE);
+        }
+      }
+
+      lastLevel = property.getLevel();
+
+      printDiagnosticsNodeProperty(console, "", property, null, false);
+    }
+
+    console.print(errorSeparator + "\n\n", TITLE_CONTENT_TYPE);
+  }
+
+  private void printDiagnosticsNodeProperty(ConsoleView console, String indent, DiagnosticsNode property,
+                                            ConsoleViewContentType contentType,
+                                            boolean isInChild) {
+    // TODO(devoncarew): Change the error message display in the framework.
+    if (property.getDescription() != null && property.getLevel() == DiagnosticLevel.info) {
+      // Elide framework '◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤...' lines.
+      if (property.getDescription().startsWith("◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤")) {
+        return;
+      }
+
+      // Elide framework blank styling lines.
+      if (StringUtil.equals("ErrorSpacer", property.getType())) {
+        return;
+      }
+    }
+
+    if (contentType == null) {
+      if (property.getLevel() == DiagnosticLevel.error) {
+        contentType = ERROR_CONTENT_TYPE;
+      }
+      else if (property.getLevel() == DiagnosticLevel.summary) {
+        contentType = ERROR_CONTENT_TYPE; // NORMAL_CONTENT_TYPE;
```

Comment:
nit: remove commented out line. Perhaps change ERROR_CONTENT_TYPE to ERROR_SUMMARY_TYPE

Reply:
👍